src/com/lftechnology/java/training/dipak/multiples/Multiples.java

```diff
@@ -0,0 +1,104 @@
+package com.lftechnology.java.training.dipak.multiples;
+
+import java.util.Scanner;
+
+import java.util.logging.Logger;
+
+/*	
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+
+ *Find the sum of all the multiples of 3 or 5 below 1000.
+*/
+
+public class Multiples {
+	private final static Logger LOGGER = Logger.getLogger(Multiples.class.getName());
+	int multiple[];	
+	int index=0;
+	int range;		
+	Multiples(int a){
+		range=a;
+		multiple=new int[range];	
+	}
+	
+	/**
+	 * <p>This method check all the multiples of the user defined numbers that are taken as input within the user
+	 * defined range </p>
+	 * @param  array a[]

```

Comment:
Don't need to include square braces " [ ] " if you have stated the data type, i.e array.